src/test/java/nextstep/subway/unit/TokenAuthenticationInterceptorTest.java

```diff
@@ -30,19 +30,32 @@
     private static final String EMAIL = "email@email.com";
     private static final String PASSWORD = "password";
 
-    private CustomUserDetailsService customUserDetailsService;
+    private UserDetailsService userDetailsService;
+    private UserDetails userDetails;
     private JwtTokenProvider jwtTokenProvider;
     private TokenAuthenticationInterceptor interceptor;
 
     @BeforeEach
     void setUp() {
-        customUserDetailsService = mock(CustomUserDetailsService.class);
+        userDetailsService = mock(UserDetailsService.class);
         jwtTokenProvider = mock(JwtTokenProvider.class);
+        userDetails = new UserDetails() {
```

Comment:
```suggestion
        userDetails = credentials -> true;
```

조금 더 간략한 구현이 가능할 것 같아요 😄